support-frontend/knip.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 	// ! signals that this is production code https://knip.dev/features/production-mode
 	entry: [...flattenedEntryPoints, 'scripts/build-ssr-content.tsx!'],
 	project: ['**/*.{js,jsx,ts,tsx,scss}!'],
-	ignoreExportsUsedInFile: true,
+	ignoreExportsUsedInFile: false,
```

Comment:
Ah the reason all these `export`s are removed and this will pick it up going forwards?